### PR TITLE
PP-5398 Show transaction template and handler

### PR DESCRIPTION
**Pull request by sfount**
* get gateway account and service to compliment transaction data
* show transaction template for payments

### CODE REVIEW

File: src/assets/payuk-toolbox.scss [hunk 1]
```diff
@@ -77,3 +77,55 @@
...
+  letter-spacing: inherit;
+  margin-left: 5px;
+}
+
+.status-inline-spacer {
+  margin-bottom: 15px;
+}
+
+.status-spacer {
+  margin-bottom: 40px;
```
Comment (jankowiakmaria) [inline src/assets/payuk-toolbox.scss]: Shouldn't we introduce variables to have standard margins/paddings/borders...?

File: src/web/modules/transactions/payment.njk [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        <tr class="govuk-table__row">
+          <th class="govuk-table__cell payment__cell" scope="row"><span class="govuk-caption-m">Description</span></th>
+          <td class="govuk-table__cell payment__cell">{{ transaction.description }}</td>
+        </tr>
+        <tr class="govuk-table__row">
+          <th class="govuk-table__cell payment__cell" scope="row"><span class="govuk-caption-m">Amount</span></th>
+          <td class="govuk-table__cell payment__cell">
+            <span>{{ transaction.total_amount | currency }} </span>
+            {% if transaction.corperate_surcharge %}
+            <span>(including {{ transaction.corperate_surcharge | currency }}) corperate surcharge</span>
```
Comment (jankowiakmaria) [inline src/web/modules/transactions/payment.njk]:
```suggestion
            <span>(including {{ transaction.corporate_surcharge | currency }}) corporate surcharge</span>
```

File: src/web/modules/transactions/payment.njk [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+          <th class="govuk-table__cell payment__cell" scope="row"><span class="govuk-caption-m">Provider transaction</span></th>
+          <td class="govuk-table__cell payment__cell">{{ transaction.gateway_transaction_id }}</td>
+        </tr>
+        <tr class="govuk-table__row">
+          <th class="govuk-table__cell payment__cell" scope="row"><span class="govuk-caption-m">Delayed capture</span></th>
+          <td class="govuk-table__cell payment__cell">{{ transaction.delayed_capture | string | capitalize }}</td>
+        </tr>
+        <tr class="govuk-table__row">
+          <th class="govuk-table__cell payment__cell" scope="row"><span class="govuk-caption-m">Authorise duration</span></th>
+          <td class="govuk-table__cell payment__cell">20ms</td>
```
Comment (jankowiakmaria) [inline src/web/modules/transactions/payment.njk]: 😄

File: src/web/modules/transactions/transactions.http.ts [hunk 4]
```diff
@@ -2,14 +2,15 @@
...
-import { Ledger } from '../../../lib/pay-request'
+import { Ledger, Connector, AdminUsers } from '../../../lib/pay-request'
 import * as logger from '../../../lib/logger'
 
 export async function show(req: Request, res: Response, next: NextFunction): Promise<void> {
   try {
     const transaction = await Ledger.transaction(req.params.id) as Transaction
-    logger.info(`Fetched transaction ${transaction.charge_id} from Ledger`)
-    res.status(200).send()
+    const account = await Connector.account(transaction.gateway_account_id)
```
Comment (jankowiakmaria) [inline src/web/modules/transactions/transactions.http.ts]: I might be blind but is it used anywhere in the view?

File: src/web/modules/transactions/payment.njk (hunk 2 above)
Comment (sfount) [reply src/web/modules/transactions/payment.njk]: Good catch! will fix as part of cleanup

File: src/web/modules/transactions/payment.njk (hunk 3 above)
Comment (sfount) [reply src/web/modules/transactions/payment.njk]: this placeholder is gone now!

**Pull request merged**